new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+
+
+# Lesson 1.4: TEXT EDITING: INSERTION
+
+** Press  «i» and «I» to insert text. **
+
+  1. Move the cursor to the first line below marked --->.
+
+  2. To make the first line the same as the second, move the cursor on top
+     of the first character AFTER where the text is to be inserted.
+
+	 Tip: use the «w» motion you learned before to jump to the beginning of the
+	      word where you want to insert.
+
+  3. Press  [«i»](i)  and type in the necessary additions.
+
+  4. As each error is fixed press <Esc> to return to Normal mode.
+     Repeat steps 2 through 4 to correct the sentence.
+
+---> There is text misng this . {expect:There is some text missing from this line.}
+---> There is some text missing from this line. {expect:There is some text missing from this line.}
+
+  5. You can use the upper-case version «I» to insert text at the beginning of
+     the line regardless of where on the line your custor is. Place the cursor
+	 anywhere on the line below and hit «I» to insert.
+
+---> is some text missing from this line. {expect:There is some text missing from this line.}
+---> There is some text missing from this line. {expect:There is some text missing from this line.}
+
+The «i» operator does not take any count or motion, it drops us straight into
+insert mode. There are actually operators that can take a cound and motion
+before transitioning into insert mode, but they wouldn't make sense here.
+
+# Lesson 1.5: TEXT EDITING: APPENDING
+
+** Press  «a» and «A»  to append text. **
+
+  1. Move the cursor to the first line below marked --->.
+
+  2. To make the first line the same as the second, move the cursor on top
+     of the first character BEFORE where the text is to be inserted.
+
+	 Tip: use the «e» motion you learned before to jump to the end of the word
+	 where you want to append.
+
+  3. As the text has been appended press <Esc> to return to Normal mode.
+
+---> There is text misng this. {expect:There is some text missing from this line.}
+---> There is some text missing from this line. {expect:There is some text missing from this line.}
+
+  5. You can use the upper-case version «A» to append text at the end of the
+     line regardless of where on the line your custor is. Place the cursor
+     anywhere on the line below and hit «A» to append.
+
+---> There is some text missing from th {expect:There is some text missing from this line.}
+---> There is some text missing from this line. {expect:There is some text missing from this line.}
+
+Appending is very similar to inserting, except it appends text after the cursor.
+You will use both commands frequently, so you should get into the habit of using
+both when they are appropiate. Don't use «li» (move left, then insert) to append
+text, that is slower and less repeatable than appending directly.  One core idea
+in Vim is to do tasks in as few steps as possible, which can then be automated.
+Get early into the habit of being lazy.
+
+# Lesson 6.1: THE OPEN COMMAND
```

Comment:
`6.1` -> `1.6`, and maybe it could be `Lesson 1.6: TEXT EDITING: OPENING` for consistency with the other titles.